src/averageAges.js

```diff
@@ -14,48 +14,51 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const filteredMen = century === 0
+    ? people.filter(person => (
+      person.sex === 'm')
+    )
+    : people.filter(person => (
+      person.sex === 'm' && Math.ceil(person.died / 100) === century
```

Comment:
Pay attention to comments
```suggestion
      person.sex === 'm' 
      && Math.ceil(person.died / 100) === century
```